src/app/multi-line-chart/multi-line-chart.component.ts

```diff
@@ -36,12 +37,13 @@ export class MultiLineChartComponent implements OnInit {
       const columns = data.columns.slice(1);
       return {
         y: this.yAxisLabel,
+        x: this.xAxisLabel,
         series: this.data,
         // data.map(d => ({
         //   name: d.name.replace(/, ([\w-]+).*/, ' $1'),
         //   values: columns.map(k => +d[k]),
         // })),
-        dates: this.data[0].values.map((_, index) => index),
+        dates: this.data[1].values.map((_, index) => index),
```

Comment:
optional: instead of hardcoding `1`, compute which of the series has the most data points

non-optional: if you don't do the above, just write a TODO comment lol (it's obvious right now where this `1` comes from, but easy to forget later)

Reply:
I'll just do the right thing. (sigh)